Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -47,6 +48,8 @@ public struct Configuration: Equatable {
     public let rules: [Rule]
     public var rootPath: String?              // the root path to search for nested configurations
     public var configurationPath: String?     // if successfully loaded from a path
+    public var hash: Int?
```

Comment:
Mainly because I thought it'd be too much work to make `Rule` conform to `Hashable` as well. Do you think it's worth the extra complexity?